### PR TITLE
fix(axios-extension): Change log levels to debug so they dont pollute CLIs.

### PR DESCRIPTION
**Pull request by IainSAP**
https://github.com/SAP/open-ux-tools/issues/2270

Fix for CLI pollution (axios-extension):

image

- Change info to debug as info is not appropriate for a low level networking module that is consumed by CLI prompts, and there is no way to change the default level or disable.

### CODE REVIEW

File: packages/axios-extension/src/auth/index.ts [hunk 1]
```diff
@@ -97,7 +97,7 @@ export function getReentranceTicketAuthInterceptor({
         if (apiUrl && apiUrl != provider.defaults.baseURL) {
             // Reentrance tickets work with API hostnames. If the original URL was not one, this will replace it
             // with the API hostname returned
-            provider.log.warn(
+            provider.log.debug(
```
Comment (IainSAP) [inline packages/axios-extension/src/auth/index.ts:100]: Should not be a warning, the end user cannot do anything about this!

File: packages/axios-extension/src/abap/message.ts [hunk 2]
```diff
@@ -120,13 +120,13 @@ function logFullURL({
         const base = new URL(host).origin; // We only care for the origin value
         // Add this instruction to the user because of this bug in VS Code: https://github.com/microsoft/vscode/issues/144898
         // It undoes the encoding of special characters in the URL sent to the browser
-        log.info('Please copy/paste this URL in a browser for more details:');
+        log.debug('Please copy/paste this URL in a browser for more details:');
```
Comment (IainSAP) [inline packages/axios-extension/src/abap/message.ts:123]: Should not be end user messaging in a low level module. The consumer needs to deal with this directly based on a returned message.

**Comment by IainSAP** [comment]
The correct approach is to support log configuration (log level, or a new logger instance) so consumers can set the log level to off or whatever is appropriate for their usage. But for now we need to quickly hide them.

**Review by cianmSAP — APPROVED** [review]
Thanks @IainSAP 

Changes look good
Tests updated
Changeset ✅

**Review by cianmSAP — APPROVED** [review]
Re-approving

File: packages/axios-extension/src/abap/message.ts (hunk 2 above)
Comment (devinea) [reply packages/axios-extension/src/abap/message.ts:123]: Makes sense. But better to keep this as info for now until that is addressed?

File: packages/axios-extension/src/abap/message.ts [hunk 3]
```diff
@@ -120,13 +120,13 @@ function logFullURL({
...
         if (isDest) {
-            log.info(
+            log.debug(
```
Comment (devinea) [inline packages/axios-extension/src/abap/message.ts:125]: same

File: packages/axios-extension/src/abap/message.ts [hunk 4]
```diff
@@ -120,13 +120,13 @@ function logFullURL({
...
                 '(Note: You will need to replace the host in the URL with the internal host, if your destination is configured using an On-Premise SAP Cloud Connector)'
             );
         }
-        log.info(new URL(path, base).href);
+        log.debug(new URL(path, base).href);
```
Comment (devinea) [inline packages/axios-extension/src/abap/message.ts:129]: same

File: packages/axios-extension/src/abap/ui5-abap-repository-service.ts [hunk 5]
```diff
@@ -207,7 +207,7 @@ export class Ui5AbapRepositoryService extends ODataService {
                 const query = this.defaults.params?.['sap-client']
                     ? '?sap-client=' + this.defaults.params['sap-client']
                     : '';
-                this.log.info(`App available at ${frontendUrl}${path}${query}`);
+                this.log.debug(`App available at ${frontendUrl}${path}${query}`);
```
Comment (devinea) [inline packages/axios-extension/src/abap/ui5-abap-repository-service.ts:210]: same

File: packages/axios-extension/src/abap/message.ts [hunk 6]
```diff
@@ -85,7 +85,7 @@ export function prettyPrintMessage({
 }): void {
     try {
         const jsonMsg = JSON.parse(msg) as SuccessMessage;
-        log.info(jsonMsg.message);
+        log.debug(jsonMsg.message);
```
Comment (devinea) [inline packages/axios-extension/src/abap/message.ts:88]: This is the message from the server to show to the user? Should still be info then.

File: packages/axios-extension/src/abap/ui5-abap-repository-service.ts [hunk 7]
```diff
@@ -381,7 +381,7 @@ export class Ui5AbapRepositoryService extends ODataService {
                 // We've nothing to return as we dont want to show the exception to the user!
                 return Promise.resolve(undefined);
             } else {
-                this.log.info(`${appName} found on target system: ${isExisting}`);
+                this.log.debug(`${appName} found on target system: ${isExisting}`);
```
Comment (devinea) [inline packages/axios-extension/src/abap/ui5-abap-repository-service.ts:384]: same

File: packages/axios-extension/src/abap/message.ts (hunk 6 above)
Comment (IainSAP) [reply packages/axios-extension/src/abap/message.ts:88]: End users do not use this module directly and so all messages should be returned for display in the appropriate way depending on the UI environment. Im curious how an end user would ever see these messages from YUI or VSIX?

**Pull request closed without merging**